src/handlebars/archive.handlebars

```diff
@@ -1,4 +1,4 @@
-{{> header title='Archive | ' }}
+{{> header title='Archive | ' style1='<link rel="stylesheet" href="http://pagination.js.org/dist/2.0.7/pagination.css">' }}
```

Comment:
hmm maybe it's overkill - I'm just concerned we'll lag in the libs that we're pulling in.